p2pv2/type.go

```diff
@@ -6,6 +6,66 @@ import (
 	p2pPb "github.com/xuperchain/xuperunion/p2pv2/pb"
 )
 
+// CorePeersInfo defines the peers' info for core nodes
+// By setting this info, we can keep some core peers always connected directly
+// It's useful for keeping DPoS key network security and for some BFT-like consensus
+type CorePeersInfo struct {
+	Name           string   // distinguished name of the node routing
+	CurrentTermNum int64    // the current term number
+	CurrentPeerIDs []string // current core peer IDs
```

Comment:
Does p2p needs to know the term info?

Reply:
It's better to know the current term number for implementing BFT-like consensus, but I still not found a good way to tell the accurate  term info to p2p. Currently no use.